genrules.cpp

```diff
@@ -2501,8 +2501,51 @@ int Game::GenRules(const AString &rules, const AString &css,
 		temp += "these items are worth more when sold in towns, so selling "
 			"them and using the money is more economical than using them for "
 			"maintenance.";
-	};
-	f.Paragraph(temp);
+		f.Paragraph(temp);
+
+		temp = "In summary, the order of priority for how the maintenance fee is paid "
```

Comment:
I think `Here is how the maintenance fee is paid:` would be better. But still, there is the passive voice...